airflow/providers/amazon/aws/hooks/datasync.py

```diff
@@ -52,7 +52,7 @@ class AWSDataSyncHook(AwsBaseHook):
     TASK_EXECUTION_FAILURE_STATES = ("ERROR",)
     TASK_EXECUTION_SUCCESS_STATES = ("SUCCESS",)
 
-    def __init__(self, wait_interval_seconds: int = 5, *args, **kwargs) -> None:
+    def __init__(self, wait_interval_seconds: int = 30, *args, **kwargs) -> None:
```

Comment:
Change in defaults should be documented in the changelog:

https://github.com/apache/airflow/blob/main/airflow/providers/amazon/CHANGELOG.rst

Reply:
Thanks for the feedback. 
I've added it there, hopefully in the right place.

Reply:
Oh you will need to add 2.0.1 / 2.1.0 and put it under it

Reply:
@kaxil thank you. I think it's right now, please check :)